A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/PFD/A320_Neo_PFD.css

```diff
@@ -474,3 +474,7 @@ a320-neo-pfd-element #Mainframe #InstrumentsContainer #ILS {
     stroke: #ff9a00;
   }
 }
+
+.blink {
+  animation : blinking 1.3s infinite;
```

Comment:
@Kimbyeoungjang However I measure the reference you've given in the PR, I get a cycle time of 1s, not 1.3s. How did you come up with 1.3s?

Reply:
Because i check with video(i’m not pilot XD) with sim. 1sec too fast than video. So i measure time and 1.3 is better than 1sec so use that